README.md

```diff
@@ -309,16 +316,25 @@ Arguments:
   # of replica                                         Number of targeting replica of the dataset (default: 10)
 
 Options:
-  -u, --url-prefix <urlprefix>                         URL prefix for car downloading. Must be reachable by provider's boostd node. (default: "http://127.0.0.1/")
+  -u, --url-prefix <urlprefix>                         URL prefix for car downloading. Must be reachable by provider's boostd node.
+                                                       (default: "http://127.0.0.1/")
   -p, --price <maxprice>                               Maximum price per epoch per GiB in Fil. (default: "0")
   -r, --verified <verified>                            Whether to propose deal as verified. true|false. (default: "true")
   -s, --start-delay <startdelay>                       Deal start delay in days. (StartEpoch) (default: "7")
   -d, --duration <duration>                            Duration in days for deal length. (default: "525")
   -o, --offline <offline>                              Propose as offline deal. (default: "true")
-  -m, --max-deals <maxdeals>                           Max number of deals in this replication request per SP, per cron triggered. (default: "0")
-  -c, --cron-schedule <cronschedule>                   Optional cron to send deals at interval. Use double quote to wrap the format containing spaces.
-  -x, --cron-max-deals <cronmaxdeals>                  When cron schedule specified, limit the total number of deals across entire cron, per SP.
-  -xp, --cron-max-pending-deals <cronmaxpendingdeals>  When cron schedule specified, limit the total number of pending deals determined by dealtracking service, per SP.
+  -m, --max-deals <maxdeals>                           Max number of deals in this replication request per SP, per cron triggered.
```

Comment:
try not make those line breaks. You can increase the terminal width to get the description shown in a single line

Reply:
Does our lint limits line length?

Reply:
Not sure, you can try

Reply:
Actually it does not